plugins/woo-ai/src/hooks/useStoreBranding.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { useEffect } from 'react';
+import { useQuery, UseQueryResult } from 'react-query';
```

Comment:
I'm not familiar with `react-query`, and I definitely see some benefits, although I _do_ believe we have [a data store](https://github.com/woocommerce/woocommerce/blob/d70080a444972d7115298c4fe2457ca7c5f2f09f/packages/js/data/src/settings/index.ts#L5) available to access the settings as well. There's an example of it being used [here](https://github.com/woocommerce/woocommerce/blob/d70080a444972d7115298c4fe2457ca7c5f2f09f/plugins/woocommerce-admin/client/task-lists/fills/tax/index.tsx#L56).

That would be the usual way of doing this I believe. I also see some benefit to using `react-query`, but there's also a danger of us all using different libraries/methods of managing this data. Using the data store would benefit from caching by others using the same method for these settings.

Reply:
Right on, thanks for that information @tommyshellberg . Given that there is some prior-art that definitely simplifies things. There may be some complexities such as those discussed above in combining functionality with the data-stores, although I don't think our purposes demand their own data store at this point. We can just go with something like the above for a first iteration and retool as needed.